### PR TITLE
Adds Style/Tab cop to rubocop.yml

### LINKED ISSUE #1428: Enable Style/Tab Rubocop cop
Enable Style/Tab Rubocop cop:
https://rubocop.readthedocs.io/en/latest/cops_style/#styletab :bowtie:

### PR DESCRIPTION
**Pull request by codyborders**
This cop checks syntax for hard tabs. 

Style/Tab offense exclusions are removed from rubocop_todo.yml.

Cop offenses have been corrected.

Fixes #1428

### CODE REVIEW

**Comment by BelieveC** [comment]
@codyborders Nice work! 
> Issue #1428 asks for this cop to be enabled, but is it not enabled by default?

In the PR description
Instead of just referencing the issue, Please add how you addressed the issue.

Also please also check commit messages guidelines [here](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md#commits) :wink:

**Comment by Ana06** [comment]
@codyborders 

> Issue #1428 asks for this cop to be enabled, but is it not enabled by default?

It is. But when adding Rubocop to an existing project, most likely there are thousand of offences and you don't want to enable all of them at the same time  as it is a lot of work and it will get really difficult to review it. That's why Rubocop has a feature to autogenerate a todo file to exclude files with offenses. So for this cop you have to remove it from here first:

https://github.com/openSUSE/osem/blob/master/.rubocop_todo.yml#L614

And then Rubocop will complain about the offenses, that you should also fix.

Also, read the [commit messages guidelines](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md#commits) :wink:  The commit should have a title (that it is correct in your commit), description explaining what has been done or giving some relevant details and at the end the issue it fix (`Fixes #1428`)

**Comment by codyborders** [comment]
@Ana06 Sorry, I should have labelled my PR as work in progress/not to be merged 😁 I have updated the PR message according to the guidelines. Thanks for your help!

File: .rubocop.yml [hunk 1]
```diff
@@ -100,6 +100,10 @@ Style/TrailingWhitespace:
 Style/ZeroLengthPredicate:
   Enabled: true
 
+# This cop checks for tabs where spaces should be used.
+Style/Tab:
```
Comment (Ana06) [inline .rubocop.yml]: This file is alphabetically ordered, so this should be in line 91 :wink:

**Comment by codyborders** [comment]
@Ana06 I made the changes to the file then attempted to squash my commits, but I believe I've made a mistake somewhere as the [pr now shows my commit plus that of another contributor](https://github.com/openSUSE/osem/pull/1430/commits) 🙄

**Comment by Ana06** [comment]
@codyborders you still have to order the cop alphabetically.

After that you can add this change to your last commit with:

```
git add .
git commit --amend
```

And to fix the other commit that shouldn't be in your PR, try this:

1. Add upstream (maybe you already have it)

` git remote add upstream https://github.com/openSUSE/osem.git`

2. And then rebase:

`git rebase upstream master`

3. After that push again and it should be solved. :wink:

**Comment by codyborders** [comment]
@Ana06 I think I have it fixed now! Let me know if I need to make any other changes 😁

**Pull request merged**